airflow/docker.go

```diff
@@ -105,14 +102,13 @@ func imageBuild(path, imageName string) error {
 }
 
 // generateConfig generates the docker-compose config
-func generateConfig(projectName, airflowHome string, envFile string) (string, error) {
+func generateConfig(projectName, airflowHome, envFile string) (string, error) {
```

Comment:
It seems like this linter is removing the types except for the last param.  Are the other types implicit based off the last param in the list?